### DISCUSSION TITLE
Setting cell-output-ipywidget-background from an anywidget

### DISCUSSION THREAD

**Discussion by ManonMarchand** [microsoft/vscode-jupyter#15341] [Questions and Answers]
Hi here, 

I'd like to create a widget with a transparent background, and followed https://github.com/microsoft/vscode-jupyter/issues/11423 solution. A minimal example looks like this: 

```python
import anywidget

class DivWidget(anywidget.AnyWidget):
    _esm = """
    function render({ model, el }) {
      let myDiv = document.createElement("div");
      myDiv.classList.add("my-div");
      myDiv.style.height = '200px';
      el.appendChild(myDiv);
    }
	export default { render };
    """
    _css="""
    .my-div { background-color: transparent; }
    .cell-output-ipywidget { background-color: transparent; }
    """

my_div = DivWidget()
my_div
```

Executed in a cell and with anywidget installed. 

The output is a white div. And when inspecting the css I see this: 

```
.cell-output-ipywidget-background {
    background: transparent;
}
<style>
.cell-output-ipywidget-background {
    background: white!important;
}
``` 

What did I do wrong?

EDIT: looks related to https://github.com/microsoft/vscode-jupyter/issues/7161

**Comment by superstar54** [comment]
Hi @ManonMarchand, may I ask how you inspect the CSS? I also want to debug my anywidget inside VSCode, but I can't find a way to inspect the element and show the console outputs.

**Reply by ManonMarchand** [reply]
Sure, you use the `Developer: Open Webview Developer Tools` from the command palette

**Reply by superstar54** [reply]
Thanks! It works great!

**Comment by billti** [comment]
I've just been wrestling with this. This PR may be of interest to you: https://github.com/microsoft/qsharp/pull/2120

**Reply by ManonMarchand** [reply]
Ahah yes, we could do a similar hack while waiting for a better solution ^^
Thanks!
